### DISCUSSION TITLE
Wanted to create a Dockerfile for installation of Ubuntu 22 LTS with Kairos

### DISCUSSION THREAD

**Discussion by SSWAMIN1SSS** [kairos-io/kairos#1890] [General]
Similar to the link:
https://kairos.io/docs/reference/build-from-scratch/
Wanted to install latest version of Ubuntu 22.04 with auroraboot.
Used the Dockerfile for Ubuntu from this link:
https://github.com/kairos-io/kairos/blob/master/images/Dockerfile.ubuntu 
Could anyone help me in resolving this.

**Comment by mauromorales** [comment]
@SSWAMIN1SSS can you describe where you're stuck?

**Reply by vadimzharov** [reply]
@SSWAMIN1SSS Hi! I recently did this, maybe my guide will be helpful - https://github.com/vadimzharov/k8s-kairos-deploy.

**Reply by mauromorales** [reply]
Thanks for sharing @vadimzharov ! Do you have this published on a blog? would like to share it in the newsletter if it's ok with you

**Reply by vadimzharov** [reply]
Hi Not yet, planning to publish. I'll share the link once I publish it and you share it in the newsletter.

**Reply by vadimzharov** [reply]
@mauromorales I published it on Medium, feel free to share it [https://medium.com/@vadimzharov/testing-kairos-as-an-os-for-k8s-cluster-050a4af53c3d](https://medium.com/@vadimzharov/testing-kairos-as-an-os-for-k8s-cluster-050a4af53c3d)

**Reply by mauromorales** [reply]
@vadimzharov fantastic, thanks!
